02-express-tutorial/routes/auth.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
+const router = express.Router();
+
+//POst method HTTP
+router.post("/", (req, res) => {
+    const { name } = req.body;
+      if(name){
+        return res.status(200).send(`Welcome ${name}`);    
+      } else {
+        res.status(401).send("Please provide credentials");
+      }
```

Comment:
Nitpick:
This can be simplified using the ternary operator.
```js
return req.body.name
  ? res.status(200).send(`Welcome ${name}`);
  :  res.status(401).send("Please provide credentials");
```